go/libraries/events/file_backed_proc.go

```diff
@@ -175,42 +182,42 @@ func (fbp *FileBackedProc) WriteEvents(version string, evts []*eventsapi.ClientE
 		plat = eventsapi.Platform_WINDOWS
 	}
 
-	if err := fbp.ed.MakeEventsDir(); err != nil {
-		return err
-	}
+	if dirExists := fbp.EventsDirExists(); dirExists {
+		eventsPath := fbp.ed.getPath()
+		tempFilename := filepath.Join(eventsPath, localPath)
 
```

Comment:
I was having an issue by not creating the events directory until `WriteEvents()` was called, so I changed that.
I now create the directory when I instantiate a `FileBackedProc`, and just ensure the directory exists before I write the files.